Gemfile

```diff
@@ -37,7 +37,7 @@ group :test do
   gem "rack_session_access"
   gem "rails-controller-testing"
   gem "rspec-rails", "~> 4"
-  gem "rubocop-govuk"
+  gem "rubocop-govuk", "=3.6.0"
```

Comment:
can  we use `~> 3.6` here?

Reply:
Eek, this was a mistake.  Well spotted.

Reply:
Do we just enforce in the lock file then?

Reply:
just saw the comment, all good :+1: